### PR TITLE
Remove pytorch env var

### PR DESCRIPTION
**Pull request by sallyom**
A few things I noticed when running the standalone script

/cc @MichaelClifford @leseb

### CODE REVIEW

File: standalone/standalone.py [hunk 1]
```diff
@@ -311,7 +311,7 @@ def upload_s3_file():
 
 if [ "$STRATEGY" == "download" ]; then
   # List top-level directories only (no nested directories)
-  top_level_dirs=$(tar --exclude='*/*' --list --file {data_pvc_mount_path}/data.tar.gz)
+  top_level_dirs=$(tar  -tzf {data_pvc_mount_path}/data.tar.gz | grep '/$')
```
Comment (leseb) [inline standalone/standalone.py:314]: I'm actually removing this part in another PR (will push soon). This takes as long as untarring so not ideal... Can you remove this? Thanks!

**Review by leseb — COMMENTED** [review]
Just one comment about the tar inspect

File: standalone/standalone.py (hunk 1 above)
Comment (sallyom) [reply standalone/standalone.py]: removed!

**Review by leseb — CHANGES_REQUESTED** [review]
you need to edit `standalone.tpl` then run `make standalone`. Thanks

**Comment by sallyom** [comment]
I don't see anything to update in standalone.tpl, I removed the tar update commit - ready for review

**Comment by leseb** [comment]
> I don't see anything to update in standalone.tpl, I removed the tar update commit - ready for review

My bad this part comes from a component. All good!

**Review by leseb — APPROVED** [review]

**Pull request merged**